error[E0277]: the trait bound `DataSample: From<dora_operator_api_types::safer_ffi::Vec<u8>>` is not satisfied

**Describe the bug**
I was following instructions here (https://dora.carsmos.ai/docs/guides/Installation/installing#cargo-and-pip) to install dora-rs using cargo. When executing `cargo install dora-daemon`, I encountered 
```
error[E0277]: the trait bound `DataSample: From<dora_operator_api_types::safer_ffi::Vec<u8>>` is not satisfied
   --> /home/dev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/dora-runtime-0.2.5/src/operator/shared_lib.rs:127:44
    |
127 |                 data: Some(data.to_owned().into()),
    |                                            ^^^^ the trait `From<dora_operator_api_types::safer_ffi::Vec<u8>>` is not implemented for `DataSample`
    |
    = help: the trait `From<std::vec::Vec<u8>>` is implemented for `DataSample`
    = note: required for `dora_operator_api_types::safer_ffi::Vec<u8>` to implement `Into<DataSample>`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `dora-runtime` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `dora-daemon v0.2.5`, intermediate artifacts can be found at `/tmp/cargo-installpVjJSx`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

```

**To Reproduce**
Steps to reproduce the behavior:
1. `cargo install dora-cli`
2. `alias dora='dora-cli'`
3. `cargo install dora-coordinator`
4. `cargo install dora-daemon`

**Expected behavior**
Successful installation of 'dora-daemon'

**Screenshots or Video**

![Screenshot from 2023-09-13 22-23-30](https://github.com/dora-rs/dora/assets/25329580/0ffac246-3610-4eb3-8996-c662b475f5d8)

**Environments (please complete the following information):**
 - System info: Linux dev-TensorBook-2022 5.15.0-83-generic #92-Ubuntu SMP Mon Aug 14 09:30:42 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

 - Dora version: dora-cli 0.2.5


**Additional context**
N/A

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,30 +30,30 @@ members = [
 
 [workspace.package]
 # Make sure to also bump `apis/node/python/__init__.py` version.
-version = "0.2.5"
+version = "0.2.6"
 description = "`dora` goal is to be a low latency, composable, and distributed data flow."
 documentation = "https://dora.carsmos.ai"
 license = "Apache-2.0"
 
 [workspace.dependencies]
-dora-node-api = { version = "0.2.5", path = "apis/rust/node", default-features = false }
-dora-node-api-python = { version = "0.2.5", path = "apis/python/node", default-features = false }
-dora-operator-api = { version = "0.2.5", path = "apis/rust/operator", default-features = false }
-dora-operator-api-macros = { version = "0.2.5", path = "apis/rust/operator/macros" }
-dora-operator-api-types = { version = "0.2.5", path = "apis/rust/operator/types" }
-dora-operator-api-python = { version = "0.2.5", path = "apis/python/operator" }
-dora-operator-api-c = { version = "0.2.5", path = "apis/c/operator" }
-dora-node-api-c = { version = "0.2.5", path = "apis/c/node" }
-dora-core = { version = "0.2.5", path = "libraries/core" }
-dora-tracing = { version = "0.2.5", path = "libraries/extensions/telemetry/tracing" }
-dora-metrics = { version = "0.2.5", path = "libraries/extensions/telemetry/metrics" }
-dora-download = { version = "0.2.5", path = "libraries/extensions/download" }
-shared-memory-server = { version = "0.2.5", path = "libraries/shared-memory-server" }
-communication-layer-request-reply = { version = "0.2.5", path = "libraries/communication-layer/request-reply" }
-dora-message = { version = "0.2.5", path = "libraries/message" }
-dora-runtime = { version = "0.2.5", path = "binaries/runtime" }
-dora-daemon = { version = "0.2.5", path = "binaries/daemon" }
-dora-coordinator = { version = "0.2.5", path = "binaries/coordinator" }
+dora-node-api = { version = "0.2.6", path = "apis/rust/node", default-features = false }
+dora-node-api-python = { version = "0.2.6", path = "apis/python/node", default-features = false }
+dora-operator-api = { version = "0.2.6", path = "apis/rust/operator", default-features = false }
+dora-operator-api-macros = { version = "0.2.6", path = "apis/rust/operator/macros" }
+dora-operator-api-types = { version = "0.2.6", path = "apis/rust/operator/types" }
+dora-operator-api-python = { version = "0.2.6", path = "apis/python/operator" }
+dora-operator-api-c = { version = "0.2.6", path = "apis/c/operator" }
+dora-node-api-c = { version = "0.2.6", path = "apis/c/node" }
+dora-core = { version = "0.2.6", path = "libraries/core" }
+dora-tracing = { version = "0.2.6", path = "libraries/extensions/telemetry/tracing" }
+dora-metrics = { version = "0.2.6", path = "libraries/extensions/telemetry/metrics" }
+dora-download = { version = "0.2.6", path = "libraries/extensions/download" }
+shared-memory-server = { version = "0.2.6", path = "libraries/shared-memory-server" }
+communication-layer-request-reply = { version = "0.2.6", path = "libraries/communication-layer/request-reply" }
+dora-message = { version = "0.2.6", path = "libraries/message" }
+dora-runtime = { version = "0.2.6", path = "binaries/runtime" }
+dora-daemon = { version = "0.2.6", path = "binaries/daemon" }
+dora-coordinator = { version = "0.2.6", path = "binaries/coordinator" }
 dora-ros2-bridge = { path = "libraries/extensions/ros2-bridge" }
 dora-ros2-bridge-python = { path = "libraries/extensions/ros2-bridge/python" }
 

--- a/Changelog.md
+++ b/Changelog.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.2.6 (2023-09-14)
+
+* Update dependencies to fix some security advisories by @phil-opp in https://github.com/dora-rs/dora/pull/354
+  - Fixes `cargo install dora-daemon`
+
+
 ## v0.2.5 (2023-09-06)
 
 ### Features

--- a/README.md
+++ b/README.md
@@ -73,16 +73,16 @@ For more installation guideline, check out our installation guide here: https://
 
 1. Install the example python dependencies:
 ```bash
-pip install -r https://raw.githubusercontent.com/dora-rs/dora/v0.2.5/examples/python-operator-dataflow/requirements.txt
+pip install -r https://raw.githubusercontent.com/dora-rs/dora/v0.2.6/examples/python-operator-dataflow/requirements.txt
 ```
 
 2. Get some example operators:
 ```bash
-wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.5/examples/python-operator-dataflow/webcam.py
-wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.5/examples/python-operator-dataflow/plot.py
-wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.5/examples/python-operator-dataflow/utils.py
-wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.5/examples/python-operator-dataflow/object_detection.py
-wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.5/examples/python-operator-dataflow/dataflow.yml
+wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.6/examples/python-operator-dataflow/webcam.py
+wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.6/examples/python-operator-dataflow/plot.py
+wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.6/examples/python-operator-dataflow/utils.py
+wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.6/examples/python-operator-dataflow/object_detection.py
+wget https://raw.githubusercontent.com/dora-rs/dora/v0.2.6/examples/python-operator-dataflow/dataflow.yml
 ```
 
 3. Start the dataflow

--- a/apis/python/node/dora/__init__.py
+++ b/apis/python/node/dora/__init__.py
@@ -15,7 +15,7 @@ from enum import Enum
 from .dora import *
 
 __author__ = "Dora-rs Authors"
-__version__ = "0.2.5"
+__version__ = "0.2.6"
 
 
 class DoraStatus(Enum):

